python/langsmith/utils.py

```diff
@@ -123,7 +124,9 @@ def wrapper(*args: Any, **kwargs: Any) -> Any:
     return decorator
 
 
-def raise_for_status_with_text(response: requests.Response) -> None:
+def raise_for_status_with_text(
+    response: Union[requests.Response, httpx.Response],
+) -> None:
```

Comment:
Do we need to `except httpx.HTTPError` as well here?

Reply:
Oh ya!